aws/main.tf

```diff
@@ -157,14 +163,31 @@ module "edge" {
   master_ip = "${element(module.master.local_ip_v4, 0)}"
 }
 
+#
+# The code below (from here to end) should be identical for all cloud providers
+#
+
+# set cloudflare record (optional)
+module "cloudflare" {
+  # count values can not be dynamically computed, that's why using
```

Comment:
that's why we are* using, also this can go in single line since the next lines are longer anyway